### PR TITLE
Trial using pull request - 8

### PR DESCRIPTION
**Pull request by exile2003**
Some comment

### CODE REVIEW

**Comment by Vitalii-alterego** [comment]
Some comment from Vitalii-alterego

File: src/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import FileSaver from 'file-saver';
 //import '../node_modules/jquery/dist/jquery.min.js';
 //import './js/bootstrap-filestyle.min.js';
 
+
 //import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```
Comment (Vitalii-alterego) [inline src/index.js:13]: What's the heck?

**Comment by exile2003** [comment]
Other comment

**Comment by exile2003** [comment]
???

**Pull request merged**